### ISSUE TITLE
empty-tags error with private tag

### ISSUE THREAD

**Issue by 5saviahv** [gajus/eslint-plugin-jsdoc#671]
## Expected behavior
Not sure is it bug but "empty-tags" error is shown.

## Actual behavior
Strangely error disappears if I delete empty line after `@private` tag.

## ESLint Config

```js
{
    "jsdoc/require-jsdoc": 0,
    "jsdoc/check-param-names": 2,
    "jsdoc/check-tag-names": 2,
    "jsdoc/check-types": 2,
    "jsdoc/newline-after-description": 2,
    "jsdoc/require-description-complete-sentence": 2,
    "jsdoc/require-hyphen-before-param-description": 2,
    "jsdoc/require-param-description": 2,
    "jsdoc/require-param-name": 2,
    "jsdoc/require-param-type": 2,
    "jsdoc/require-param": 2,
    "jsdoc/valid-types": 2,

  }
}
```

## ESLint sample

```js
/**
 * Create an array.
 *
 * @private
 * 
 * @param {string[]} [elem] - Elements to make an array of.
 * @param {boolean} [clone] - Optionally clone nodes.
 * @returns {string[]} The array of nodes.
 */


// Format JS code here
```

## Environment

- Node version: v12.19.0
- ESLint version 7.17.0
- `eslint-plugin-jsdoc` version: 31.0.3

##
